style.css

```diff
@@ -110,3 +138,238 @@ main {
   margin: 10px;
   list-style: none;
 }
+
+.snapshoot {
+  width: inherit;
+  height: 220px;
+  display: block;
+}
+
+.snapshoot > img {
+  width: 100%;
+  height: 100%;
+}
+
+.frame {
+  width: 236px;
+  height: 32px;
+  display: grid;
+  margin: 10px 0;
+  align-items: center;
+  grid-template-columns: auto auto auto auto auto;
+}
+
+.frame > p,
+.frame > h2 {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.frame > h2 {
+  color: #344563;
+}
+
+.frame > div {
+  background-color: #e5e5e5;
+  border-radius: 50%;
+  height: 8px;
+  width: 8px;
+}
+
+.text > p {
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 12px 0;
+}
+
+.lang {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-column-gap: 8px;
+  width: 203px;
+  height: 24px;
+  margin: 20px 0 5px 0;
+}
+
+.lang > p {
+  height: inherit;
+  display: grid;
+  background-color: #ebebff;
+  color: #6070ff;
+  border-radius: 8px;
+  align-items: center;
+  text-align: center;
+}
+
+button {
+  padding: 10px;
+  height: 48px;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  margin: 20px 0;
+}
```

Comment:
- [ ] Amazing job styling this button, it looks exactly like in the Figma design 🔥  but you should consider adding the button's actions (such as hover, pressed, disabled). They were all provided in Figma, as you can see here:

![image](https://user-images.githubusercontent.com/66967274/148630158-8a6d7a13-0ab8-4f93-a415-af70245c8e27.png)